packages/cli/src/Db.ts

```diff
@@ -28,18 +29,8 @@ import { postgresMigrations } from './databases/postgresdb/migrations';
 import { mysqlMigrations } from './databases/mysqldb/migrations';
 import { sqliteMigrations } from './databases/sqlite/migrations';
 
-export const collections: IDatabaseCollections = {
-	Credentials: null,
-	Execution: null,
-	Workflow: null,
-	Webhook: null,
-	Tag: null,
-	Role: null,
-	User: null,
-	SharedCredentials: null,
-	SharedWorkflow: null,
-	Settings: null,
-};
+export let isInitialized = false;
+export const collections = {} as IDatabaseCollections;
```

Comment:
hmmm seeing these exports rather screams for a class ^^

Reply:
For reference `isInitialized` is used only once in `Server.ts`. Other than this, usage of the module is the exact same. Lmk if you prefer the switch.